activejob/lib/active_job/railtie.rb

```diff
@@ -49,5 +49,21 @@ class Railtie < Rails::Railtie # :nodoc:
         end
       end
     end
+
+    initializer "active_job.query_comments" do
+      require "active_record/railties/query_comments"
+
+      ActiveSupport.on_load(:active_job) do
+        config_accessor :query_comments_action_filter_enabled, instance_accessor: false do
+          true
+        end
```

Comment:
Hmm this isn't available in ActiveJob - for some reason I thought it was. Will find another approach.